js/main.js

```diff
@@ -9,10 +9,15 @@ const $form = document.querySelector('#new-entry');
 
 $form.addEventListener('submit', function (event) {
   event.preventDefault();
-  const newEntry = {};
+  const newEntry = {
+    title: null,
+    photoURL: null,
+    notes: null,
+    entryId: null
+  };
   newEntry.entryId = data.nextEntryId;
```

Comment:
No need to set `entryId` to `null` here, you can just do `entryId: data.nextEntryId`